server/Core/Entities/Coach.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Core.Entities;
+
+public class Coach
+{
+    public int MembershipCardNum { get; set; }
+    public string? InstructorCategory { get; set; }
```

Comment:
Maybe we should create a separate table for instructor categories if there are known and limited?

Reply:
I had plans to do it later. I think strings are enough for now.